paddle/fluid/framework/details/multi_devices_graph_pass.cc

```diff
@@ -226,7 +221,7 @@ std::unique_ptr<ir::Graph> MultiDevSSAGraphBuilderBase::ApplyImpl(
    * Only variables should be the leaves of graph.
    */
   AddOutputToLeafOps(&result);
-  result.Erase(kGraphOps);
+  // result.Erase(kGraphOps);
```

Comment:
Why not removing this?

Reply:
It's easy to iter all ops, and I don't know why to delete this attr, I think it's useful to do sth. about the graph.